globals/utilities.php

```diff
@@ -554,3 +564,37 @@ function neve_safe_get( $url, $args = array() ) {
 			$args
 		);
 }
+
+/**
+ * Create pagination for Easy Digital Downloads Archive.
+ */
+function neve_edd_download_nav() {
+
+	global $wp_query;
+
+	$big          = 999999;
+	$search_for   = array( $big, '#038;' );
+	$replace_with = array( '%#%', '&' );
+
+	$pagination = paginate_links(
+		array(
+			'base'      => str_replace( $search_for, $replace_with, get_pagenum_link( $big ) ),
+			'format'    => '?paged=%#%',
+			'current'   => max( 1, get_query_var( 'paged' ) ),
+			'total'     => $wp_query->max_num_pages,
+			'prev_next' => true,
+		) 
+	);
+	?>
+
+	<div id="edd_download_pagination" class="navigation">
+		<?php 
+		//phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo $pagination; 
```

Comment:
We can escape this with a custom allowed tags array, similar to how we do it for default pagination here: https://github.com/Codeinwp/neve/blob/a93d3d821c4ae4c49916736adb163a6a5c223cbc/inc/views/pluggable/pagination.php#L246